google-ads-examples/src/main/java/com/google/ads/googleads/examples/hotelads/AddHotelCallout.java

```diff
@@ -0,0 +1,231 @@
+
+            // Either pass the required parameters for this example on the command line, or insert them
+            // into the code here. See the parameter class definition above for descriptions.
+            params.customerId = Long.parseLong("INSERT_CUSTOMER_ID_HERE");
+            params.campaignId = Long.parseLong("INSERT_CAMPAIGN_ID_HERE");
+            params.calloutText = "INSERT_CAMPAIGN_TEXT_HERE";
+            params.languageCode = "INSERT_LANGUAGE_CODE_HERE";
+        }
+
+        GoogleAdsClient googleAdsClient;
+        try {
+            googleAdsClient = GoogleAdsClient.newBuilder().fromPropertiesFile().build();
+        } catch (FileNotFoundException fnfe) {
+            System.err.printf(
+                "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", 
+                fnfe
+            );
+            return;
+        } catch (IOException ioe) {
+            System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+            return;
+        }
+
+        try {
+            new AddHotelCallout()
+                .runExample(
+                    googleAdsClient,
+                    params.customerId,
+                    params.campaignId,
+                    params.calloutText,
+                    params.languageCode
+                );
+        } catch (GoogleAdsException gae) {
+            // GoogleAdsException is the base class for most exceptions thrown by an API request.
+            // Instances of this exception have a message and a GoogleAdsFailure that contains a
+            // collection of GoogleAdsErrors that indicate the underlying causes of the
+            // GoogleAdsException.
+            System.err.printf(
+                "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+                gae.getRequestId()
+            );
+            int i = 0;
+            for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()){
+                System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+            }
+        }
+    }
+
+       /**
+        * Runs the example.
+        *
+        * @param googleAdsClient the Google Ads API client.
+        * @param customerId the client customer ID.
+        * @param campaignId the campaign ID.
+        * @param calloutText the text for the callout.
+        * @param languageCode the language code for the callout (IETF BCP 47 compliant).
+        * @throws GoogleAdsException if an API request failed with one or more service errors.
+        */
+        private void runExample(
+            GoogleAdsClient googleAdsClient, 
+            long customerId, 
+            long campaignId,
+            String calloutText,
+            String languageCode) {
+
+            // Creates the extension feed item.
+            String extensionFeedItemResourceName = addExtensionFeedItem(googleAdsClient, customerId, calloutText, languageCode);
```

Comment:
Not sure the max number of chars that GitHub shows.
Perhaps running google formatter like Nick said above will auto correct this if it's too long.

Reply:
Ran the formatter.